src/vanilla/store.ts

```diff
@@ -165,25 +165,38 @@ const addDependency = <Value>(
 // Batch
 //
 
-type BatchPriority = 'H' | 'M' | 'L'
+const BATCH_PRIORITY_HIGH = 'H'
+const BATCH_PRIORITY_MEDIUM = 'M'
+const BATCH_PRIORITY_LOW = 'L'
+
+type BatchPriority =
+  | typeof BATCH_PRIORITY_HIGH
+  | typeof BATCH_PRIORITY_MEDIUM
+  | typeof BATCH_PRIORITY_LOW
 
 type Batch = Readonly<{
   /** Atom dependents map */
   D: Map<AnyAtom, Set<AnyAtom>>
   /** High priority functions */
-  H: Set<() => void>
+  [BATCH_PRIORITY_HIGH]: Set<() => void>
   /** Medium priority functions */
-  M: Set<() => void>
+  [BATCH_PRIORITY_MEDIUM]: Set<() => void>
   /** Low priority functions */
-  L: Set<() => void>
+  [BATCH_PRIORITY_LOW]: Set<() => void>
 }>
 
-const createBatch = (): Batch => ({
-  D: new Map(),
-  H: new Set(),
-  M: new Set(),
-  L: new Set(),
-})
+const createBatch = (): Batch => {
+  const batch = { D: new Map() } as {
+    D: Map<AnyAtom, Set<AnyAtom>>
+    [BATCH_PRIORITY_HIGH]: Set<() => void>
+    [BATCH_PRIORITY_MEDIUM]: Set<() => void>
+    [BATCH_PRIORITY_LOW]: Set<() => void>
+  }
+  batch[BATCH_PRIORITY_HIGH] = new Set()
+  batch[BATCH_PRIORITY_MEDIUM] = new Set()
+  batch[BATCH_PRIORITY_LOW] = new Set()
+  return batch
+}
```

Comment:
I'm not super confident with it, but this seems better for minification.
<img width="1316" alt="image" src="https://github.com/user-attachments/assets/3496ca35-9fb4-47bf-831a-4e9961060aa7" />

We will be revisiting it soon with sync effect anyway, otherwise, we can revert it back.